src/buildExec.ts

```diff
@@ -32,8 +33,9 @@ const getGitService = (): string => {
 const isPullRequestFromFork = (): boolean => {
   core.info(`evenName: ${context.eventName}`);
   if (
-    `${context.eventName}` !== 'pull_request' &&
-    `${context.eventName}` !== 'pull_request_target'
+    ['pull_request', 'pull_request_target'].every(
+        (event) => event !== context.eventName,
+    )
```

Comment:
```suggestion
    !['pull_request', 'pull_request_target'].includes(context.eventName)
```

similar to below

Reply:
Ahhh yea that would be better ... i spent a hot minute looking at MDN docs trying to figure out the best way ... and all i needed was a `!` 🤦